index.js

```diff
@@ -12,7 +12,7 @@ client.once("ready", () => {
 
 commands.forEach((element) => {
   client.api
-    .applications(config.botId) //.guilds("249809874853036033")
+    .applications(config.botId).guilds("249809874853036033")
```

Comment:
In the future I think we should have configuration settings for determining if commands should be registered globally or just to the test guild. Adding an issue for this.